src/components/NavBar/styled.ts

```diff
@@ -1,7 +1,8 @@
 import styled from 'styled-components'
-import { MediaQueries } from '../../themes'
-import { StyledLink } from '../StyledText/StyledLink'
-import { Theme } from '../Theme'
+import { MediaQueries } from 'themes'
```

Comment:
one line down :arrow_down: :upside_down_face: Or is it on purpose? MediaQueries is not foreign lib :slightly_smiling_face:

Reply:
It's just a detail. It looks like a foreign lib, so I left it like this everywhere. Does it really matters so much? :smile:

Reply:
It doesn't matter :)